xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Perhaps my first comment and the one at https://github.com/xbmc/xbmc/pull/6207#discussion_r23719344 weren't clear enough now that I read it again. I didn't mean that you should make the copy ctor and operator= public, just the standard ctors, so we can do without the friend class statement. And there's no reason to leave the implementation of those out to enforce a linker error if the private keyword already prevents it from being copied, so you can remove the wall of text in the comment explaining why there's no implementation.